core/src/avm2/globals/flash/system/capabilities.rs

```diff
@@ -26,7 +27,11 @@ pub fn get_player_type<'gc>(
     let player_type = if cfg!(target_family = "wasm") {
         "PlugIn"
     } else {
-        "StandAlone"
+        //See: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/system/Capabilities.html#playerType
```

Comment:
It seems like we usually don't really refer to the documentation for specific cases like this.

Reply:
For the documentation yes, but there're some places that reference the avmplus source like this, if you think this is unnecessary, you can remove it.

Reply:
I think it should be removed, the documentation is an implicit source for most of these cases.